themes/default/content/blog/review-stacks/index.md

```diff
@@ -0,0 +1,97 @@
+
+> “At GitHub, we are constantly seeking to improve the developer experience and our partner ecosystem plays a big role in that.” said Matthew Manning, Senior Business Development Manager for GitHub Actions, “We’re excited to support the launch of Pulumi Review Stacks with GitHub because it makes it easier for teams to review and test their products before shipping.”
+
+## Configuring Review Stacks
+
+Configuring Review Stacks is a simple three-step process:
+
+1. Create a new stack and `Pulumi.<stack>.yaml` configuration file - this config will be copied into every review stack that gets created, and can even be modified within a PR
+2. Configure Deployment Settings for the stack - this specifies how to acquire source code, cloud credentials and more when deploying via Pulumi Deployments
+3. Set the `pullRequestTemplate` Deployment Setting to true - this indicates that all pull requests against this stack’s branch should reference this stack as a Review Stack template.
+
+You can use an existing stack as a Review Stack template, as long as it has Pulumi Deployment settings configured. This will cause Review Stacks to get deployed into the same cloud account. If you want to separate the cloud resources in your production stack from the resources created via Review Stacks then you can create a separate stack and template that references a different cloud account (AWS, Azure, GCP, etc).
+
+Review Stacks and Deployment Settings can be configured via the Pulumi Cloud console, the Pulumi Cloud REST API, or within a Pulumi Program using the Pulumi Cloud Resource Provider.
+
+### REST API
+
+```
+curl -i -XPOST -H "Content-Type: application/json" -H "Authorization: token $PULUMI_ACCESS_TOKEN" \
+--location "https://api.pulumi.com/api/preview/org/project/stack/deployment/settings" \
+-d '{
+  "gitHub":{
+    "pullRequestTemplate": true
+    }
+}'
+```
+
+### Pulumi Cloud Resource Provider
+
+```typescript
+import * as pulumiservice from "@pulumi/pulumiservice";
+
+const deploymentSettings = new pulumiservice.DeploymentSettings("deploymentSettings", {
+	organization: pulumi.getOrganization(),
+	project: "your project",
+	stack: "your stack",
+	github: {
+		deployCommits: true,
+		previewPullRequests: true,
+		pullRequestTemplate: true,
+		repository: "pulumi/deployment-automation",
+	},
+	sourceContext: {
+		git: {
+			branch: "refs/heads/main",
+			repoDir: "pulumi-pet-shop",
+		},
+	},
+});
+```
+
+## Collaborate in the Cloud
+
+Cloud software is a huge asset but without a repeatable and standardized process for development, the sprawl of different toolchains and processes can slow you to a screeching halt. The cloud should be an accelerant to your business, not a bottleneck. Whether you’re a team of five or an enterprise of thousands, Review Stacks and Pulumi Deployments enables you to ship with more velocity, confidence, and collaboration.
```

Comment:
I feel like the narrative here could instead be:
1. Infrastructure as Code with Pulumi enables you to define rich and repeatable cloud infrastructure environments for your projects
2. Pulumi Deployments enables you to automate the deployment of your long-lived environments (production, staging)
3. Review Stacks takes this even further, and lets you automate deployment of per-PR development environments
4. This makes it easier than ever before for *every* person in your organization to collaborate on your infrastructure and applications.

Reply:
Trying an updated conclusion that focuses on the motivation for building this. 

Totally an option to focus on advertising and selling deployments as a whole in closing, but I think we might be able to accomplish that in the CTA list. Give it a read and let me know what you think.